test/runnable/template10.d

```diff
@@ -227,6 +227,7 @@ void test5()
 
 /********************************************/
 // inline tests
+// http://issues.dlang.org/show_bug.cgi?id=14730
```

Comment:
You should remove the `-inline` at the top of the file if you want to use it as a test for this change.

Reply:
Doesn't `// PERMUTE_ARGS: -inline`  mean that the test will be compiled once with and a second without `-inline` ?

Reply:
Indeed, I thought it uses `REQUIRED_ARGS`.